SU2_CFD/include/fluid/CIncIdealGasPolynomial.hpp

```diff
@@ -67,7 +67,7 @@ class CIncIdealGasPolynomial final : public CFluidModel {
    * \brief Set the Dimensionless State using Temperature.
    * \param[in] t - Temperature value at the point.
    */
-  void SetTDState_T(su2double t) override {
+  void SetTDState_T(su2double t, su2double *val_scalars = nullptr) override {
```

Comment:
Are the scalars input or output? input should be const.
```suggestion
  void SetTDState_T(su2double t, const su2double* val_scalars = nullptr) override {
```

Reply:
done